engine/config/1.0/schema.json

```diff
@@ -446,27 +446,30 @@
           "type": "object",
           "description": "Configures the persistent storage system.",
           "properties": {
-            "embedded": {
+            "in_memory": {
```

Comment:
Reverted all the changes to `1.0/schema.json` since we are using `2.0` to break backwards compatibility. I think we want to keep 1.0 where it is - lmk if I'm missing something.